<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateFirmwareGui</class>
 <widget class="QDialog" name="UpdateFirmwareGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>固件升级</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_close">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>升  级</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_log">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_close</sender>
   <signal>accepted()</signal>
   <receiver>UpdateFirmwareGui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_close</sender>
   <signal>rejected()</signal>
   <receiver>UpdateFirmwareGui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
